testfiles/fontload-variable.lvt

```diff
@@ -0,0 +1,33 @@
+\input{fontspec-testsetup.tex}
+
+\usepackage{fontspec}
+\usepackage{iftex}
+\ifluahbtex
+  \defaultfontfeatures{Renderer=HarfBuzz}
+\fi
```

Comment:
The feature generally also works for the `Node` renderer, but the HarfBuzz supports is a bit more consistent IMO. (I'm probably biased though)